recipes/xtr/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.1)
+project(PackageTest CXX)
+
+set(CMAKE_CXX_STANDARD 20)
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(example example.cpp)
+target_link_libraries(example ${CONAN_LIBS})
```

As part of going to conan 2.0, the use of the `cmake` generator to provide variables (in this case `${CONAN_LIBS}`) is phased out.
So use targets instead.
```suggestion
conan_basic_setup(TARGETS)

find_package(xtr REQUIRED)

add_executable(${PROJECT_NAME} example.cpp)
target_link_libraries(${PROJECT_NAME} PRIVATE xtr::xtr)
```